src/PersistentLoggedStatement.php

```diff
@@ -107,37 +137,46 @@ public function execute(array $inputParameters = null) : bool
 
     public function setFetchMode(int $mode, mixed ...$args) : bool
     {
-        return $this->parent->setFetchMode(...func_get_args());
+        return $this->parent->setFetchMode($mode, ...$args);
     }
 
     public function fetch(
-        int $fetch_style = null,
+        int $fetch_style = PDO::FETCH_DEFAULT,
         int $cursor_orientation = PDO::FETCH_ORI_NEXT,
         int $cursor_offset = 0
     ) : mixed
     {
-        return $this->parent->fetch(...func_get_args());
+        return $this->parent->fetch($fetch_style, $cursor_orientation, $cursor_offset);
     }
 
     public function fetchAll(
-        int $fetch_style = PDO::FETCH_BOTH,
+        int $fetch_style = PDO::FETCH_DEFAULT,
```

Comment:
This also looks like a BC break ... ?

Reply:
I was following the tests on this. If `FETCH_BOTH` is set then the ` Atlas\Pdo\PersistentLoggedStatementTest::testFetchAll` test fails (because there are more data than what the test checks for). We can certainly adjust the test and leave this as it was.